client/my-sites/posts/post-controls/post-control.jsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classNames from 'classnames';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+
+export const PostControl = ( { control } ) =>
+	<li className={ classNames( { 'post-controls__disabled': control.disabled } ) } >
```

Comment:
Annnnnd... in the middle of the post controls style there's a TODO that proposes to move the "more post controls" logic out of `Posts` and into `PostControls`, making it a stateful component as it was before!